client/src/index.ts

```diff
@@ -15,10 +15,6 @@ export interface XataRecord {
 }
 
 export abstract class Repository<T extends XataRecord> extends Query<T> {
-  select<K extends SelectableColumn<T>>(columns: K[]) {
-    return new Query<T, Select<T, K>>(this.repository, this.table, { columns });
-  }
```

Comment:
``Select`` is actually a method from ``Query`` which ``Repository`` extends. Moving to its correct place